app/models/state_file_base_intake.rb

```diff
@@ -122,6 +122,14 @@ def ask_spouse_name?
     filing_status_mfj?
   end
 
+  def ask_spouse_esign?
+    filing_status_mfj? && !spouse_deceased?
+  end
+
+  def spouse_deceased?
+    direct_file_data.surviving_spouse?
+  end
```

Comment:
Not a blocker:
Should we use unified terminology? (ie: `surviving_spouse?` vs `spouse_deceased?`)
Personally, I find `spouse_deceased?` to be clearer.